functions/slack.js

```diff
@@ -0,0 +1,33 @@
+const axios = require('axios');
+const functions = require('firebase-functions');
+const { IncomingWebhook } = require('@slack/webhook');
+
+exports.notifySlack = (msg) => {
+  const url = functions.config().slack_integration.webhook_url;
+  const webhook = new IncomingWebhook(url, {});
+
+  return webhook
+    .send({
+      text: msg,
+    })
+    .then(() => {
+      console.log('notification sent');
+    })
+    .catch((e) => {
+      console.log('error while posting to slack', e);
+    });
+};
+
+exports.inviteSlack = async (email) => {
```

Comment:
as far as I can see, this is not invoked anywhere, am I right?

Reply:
yes, this is *how* it can be done, I didn't touch any of the logic, just testing the function out

Reply:
are you using this now?